### PR TITLE
add responsive and fix elements style

### PR DESCRIPTION
**Pull request by fuentesloic**
Screenshot_2019-12-08 Around you
Screenshot_2019-12-08 Digital Nomad Community
Capture d’écran, le 2019-12-08 à 21 53 44
Capture d’écran, le 2019-12-08 à 21 53 53
Capture d’écran, le 2019-12-08 à 21 54 00

### CODE REVIEW

File: static/css/around-map.css [hunk 1]
```diff
@@ -2,10 +2,9 @@
 
 #open-form {
   position: absolute;
-  top: 0;
   left: 0;
   position: absolute;
-  top: 6rem;
+  bottom: 0rem;
```
Comment (vinyll) [inline static/css/around-map.css]: `bottom: 0;` Units are useless in that case

File: static/css/main.css [hunk 2]
```diff
@@ -189,7 +189,7 @@ body > footer a img {
 
 /* Main */
 main {
-  margin: 10rem 0;
+  margin: 10rem 0 0;
```
Comment (vinyll) [inline static/css/main.css]: I guess we can explicitly say `margin-top: 10rem;` in this case

File: static/css/main.css [hunk 3]
```diff
@@ -318,21 +318,22 @@ button:hover,
...
+  right: 0;
   background: rgba(50, 50, 50, .8);
   color: white;
-  padding: 1.9rem;
+  padding: 1rem 2rem;
+  margin: 2rem;
   box-shadow: 0 0 .3rem #ccc;
   transition: .3s;
   border-bottom: .1rem solid #333;
+  border-radius: 3rem;
```
Comment (vinyll) [inline static/css/main.css]: Isn't this too rounded?

**Review by vinyll — DISMISSED** [review]
Approved with comments

File: static/css/around-map.css (hunk 1 above)
Comment (fuentesloic) [reply static/css/around-map.css]: addressed with 36763544bb3d5db9544dc83e263cc92dbc78c739

File: static/css/main.css (hunk 2 above)
Comment (fuentesloic) [reply static/css/main.css]: addressed with 36763544bb3d5db9544dc83e263cc92dbc78c739

**Review by vinyll — APPROVED** [review]

**Pull request merged**